hooks/post_gen_project.py

```diff
@@ -17,15 +17,47 @@ def print_log(message):
     print(CYELLOW + message + CEND)
 
 
+def remove_heroku_files():
+    """
+    Removes heroku related files
+    """
+    shutil.rmtree(os.path.join(
+        PROJECT_DIRECTORY, "deploy/heroku"
+    ))
+    os.remove(os.path.join(
+        PROJECT_DIRECTORY, ".github/workflows/deploy.yml"
+    ))
+
+
+def remove_empty_folders():
```

Comment:
@hoangmirs Why do we need this function?

Reply:
In `remove_heroku_files` function, I just remove `deploy/heroku`, so it still has the empty `deploy` folder.
I had planned to remove the entire `deploy` folder, but I think we might have another one in that folder, so I went with this way